PGM/src/main/java/tc/oc/pgm/development/MapDevelopmentCommands.java

```diff
@@ -205,7 +200,10 @@ public void mapFeaturesCommand(CommandContext args, CommandSender sender) throws
         final Optional<String> typeFilter = CommandUtils.flag(args, 't').map(String::toLowerCase);
         final Optional<String> idFilter = CommandUtils.flag(args, 'i').map(String::toLowerCase);
 
-        Stream<? extends FeatureDefinition> features = map.getContext().features().all();
+        MapModuleContext context = map.getContext().orElseThrow(() ->
+                new CommandException("The map modules are currently unloaded."));
```

Comment:
This should never happen as the map input should always be the currently playing map

Reply:
Use a TranslatableCommandException instead, just to make stuff easier to translate if we ever consider them in the future.

Reply:
Use TranslatableCommandException instead, just to make stuff easier to translate in the future.

Reply:
This isn't an error that should be translated, this is an error that should literally never happen. If PGM has unloaded the modules of the currently running match, PGM is literally falling apart. Maybe i should switch it to an IllegalStateException

Reply:
An illegal state exception should do.